### ISSUE TITLE
Proper usage of dynamic NuxtLinks

### ISSUE THREAD

**Issue by TheFoolishPupil** [nuxt/nuxt.js#9120]
I don't believe this is a bug, but rather just an issue with how I am trying to use the slugs from store data to generate a list of links to nuxt/content files. I have the following component:
```vue
<template lang="pug">
div
  ul
    li(v-for='post in posts', :key='post.slug')
      NuxtLink(:to='post.slug') {{ post.title }}
</template>

<script>
export default {
  computed: {
    posts() {
      return this.$store.state.posts
    },
  },
}
</script>
```
It works as expected; the issue being that I'm not sure how to prefix the slug with a '/' character in the NuxtLinks `to` atribute. This results in the router trying to take me to f.ex `/first-post/first-post` if i do a browser refresh while on the page. Additionally it also drops the nuxt-active-link class from the post that I am currently looking at. 

I was able to deduce this was the issue after recreating the behavior with the default nuxt project form `yarn create nuxt-app` with a simple 2 page application with links between the two pages. I simply omitted the '/' character in the `to` attribute and see the same behavior.

Long story short, it seems my dynamic links are of the form `NuxtLink(:to='about')` as opposed to `NuxtLink(:to='/about')`

**Comment by TheFoolishPupil** [comment]
Thanks Daniel, that seems to have done the trick. I find it a little difficult to know when to use any combination of prime-marks, single quotes, double quotes, ${}, etc... If you had any tips for becoming more accustom to these patterns it would be much appreciated.

Regardless, thanks for solving my issue.

**Comment by danielroe** [comment]
@TheFoolishPupil If you set a prop without the `:`, the value you pass is a string. If you use the `:`, it's shorthand for `v-bind:` and the value passed is a JavaScript expression (number, boolean, etc). [More here](https://vuejs.org/v2/guide/syntax.html#Using-JavaScript-Expressions). So to append a `/`, you just need to do what you would do in JavaScript, whether that's using template literals or just `'/' + item.slug`.
